addon/components/chart-component.js

```diff
@@ -52,6 +52,22 @@ export default Ember.Component.extend(ColorableMixin, ResizeHandlerMixin, {
   }),
 
   /**
+   * An array of the values which are at least 0
+   * @type {Array<Number>}
+   */
+  positiveValues: Ember.computed('allFinishedDataValues.[]', function() {
+    return this.get('allFinishedDataValues').filter((val) => val >= 0);
+  }),
```

Comment:
It can't, because `allFinishedDataValues` isn't an `Ember.Array`. I'd rather not fix that in this PR. Definitely something we should do though